recipes/cyrsoxs/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "cyrsoxs" %}
+{% set version = "1.1.5.0" %}
+{% set sha = "7ce6e64d5e794751476e6e1b6dbb1cd940b2dd26796b8d416fd82b6c20302ff9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/usnistgov/cyrsoxs/archive/refs/tags/V{{ version }}.tar.gz
+  sha256: {{ sha }}
+
+build:
+  skip: True  # [py<37]
+  skip: True  # [not linux64]
+  skip: True  # [cuda_compiler_version == "None"]
+  number: 0
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+    - {{ compiler('cuda') }}
+    - cmake
+    - make
+    - openmp
```

Comment:
```suggestion
```
i also don't think you need in build. build should have, under normal circumstances, only build tools, not compilation libraries.

Reply:
Maybe see https://conda-forge.org/docs/maintainer/knowledge_base.html#openmp

Reply:
The KB article suggests putting `libgomp` in `build`, just pushed this change.